### ISSUE TITLE
[Bug]: Miru AppImages breaks upon trying to update

### ISSUE THREAD

**Issue by Hexality** [ThaUnknown/miru#481]
### Preflight checklist

- [X] I have searched the [issue tracker](https://github.com/ThaUnknown/miru/issues) for a bug report that matches the one I want to file, without success.
- [X] I have searched the [frequently asked questions](https://github.com/ThaUnknown/miru/blob/master/docs/faq.md) for a solution that fixes this problem, without success.
- [X] I have checked that I'm using the [latest stable](https://github.com/ThaUnknown/miru/releases/latest) version of the app.

### What app version are you using?

v5.2.3

### What operating system version are you using?

Linux

### Expected Behavior

Updates the AppImage

### Actual Behavior

Shows the update notification then nukes itself upon restarting, deleting the .AppImage file and breaking the app.

Also, please add an option to disable auto-updates, as some people (like me) uses softwares such as Gearlever to manage and update them and having auto-updates most of the time break the software.

### Screenshots

![image](https://github.com/user-attachments/assets/6806cd1c-30ca-4853-b442-ee672c90737d)

**Comment by Hexality** [comment]
also, before it would just nuke itself and I could redownload and run, but after the update before the last, it started loopnuking itself.
You download it again, it deletes itself and shows the same error upon launch.

**Comment by ThaUnknown** [comment]
Yeah this is annoying, and odd? Miru tells macOS to update the app, and macOS does nothing, idfk, why is it so hard to get macOS to do anything, fucks sake

**Comment by Hexality** [comment]
> Yeah this is annoying, and odd? Miru tells macOS to update the app, and macOS does nothing, idfk, why is it so hard to get macOS to do anything, fucks sake

Also happens on macOS? maybe its something mutual between *nix systems then(?)

**Comment by ThaUnknown** [comment]
oh shit, i thought this was a mac issue, yeah idk about linux, it should just work? that just looks like some corruption during update

**Comment by Hexality** [comment]
I think the only fix for this would be relying on a insteance check to see if the app is running from a temporary volume, this would probably also benefit macOS. In the case of macOS I don't know how it handles self-updates from the app but if possible you could add something like a binary update instead or, like peppy did with osu, a notification with a onclick action directing the user to the https://github.com/ThaUnknown/Miru/releases/latest/

**Comment by ThaUnknown** [comment]
I don't implement my own auto updater, i use electron's auto updater, so it's likely smth to bring up with them

**Comment by ThaUnknown** [comment]
.14 will fix this, you'll need to wait till .15 to test

**Comment by Technetium1** [comment]
I've tested the last 3 appimage auto-updates, and they have been fine. I believe this can be closed now.
